helm_deploy/hmpps-support-additional-needs-api/values.yaml

```diff
@@ -32,6 +32,8 @@ generic-service:
     hmpps-support-additional-needs-api-client-creds:
       API_CLIENT_ID: 'API_CLIENT_ID'
       API_CLIENT_SECRET: 'API_CLIENT_SECRET'
+      PRISONER_SEARCH_API_CLIENT_ID: "PRISONER_SEARCH_API_CLIENT_ID"
```

Comment:
This client has been setup by the HAAR team and added to the namespace secrets (dev only at this point)